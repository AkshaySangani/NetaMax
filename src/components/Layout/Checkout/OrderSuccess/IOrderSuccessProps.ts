import { ICoupon } from "dataflows/Checkout/ICoupon";
import { INearestStore } from "dataflows/Stores/INearestStore";

export interface IOrderSuccessProps {
  /**
   * The text for the sharing discount code whatsapp message.
   * @type {string}
   */
  shareDiscountCodeText: string;

  /**
   * The on referral share click handler.
   * @type {() => void}
   */
  onReferralShareButtonClick: () => void;

  /**
   * Shows the CoinAward card
   * @type {boolean}
   */
  showCoinAwardedCard: boolean;

  /**
   * The awarded amount
   * @type {number}
   */
  priceAmount?: number;

  /**
   * Formatted Phone number
   * @type {string}
   */
  formattedPhone: string;

  /**
   * The partial price of the items in the cart
   * @type {number}
   */
  partialSumSuccess: number;

  /**
   * The coupon sum
   * @type {number}
   */
  couponSumSuccess: number;

  /**
   * The monetas sum
   * @type {number}
   */
  monetasSumSuccess: number;

  /**
   * The total price for items in the cart
   * @type {number}
   */
  totalSumSuccess: number;

  /**
   * If pays with monetas
   * @type {boolean}
   */
  payWithMonetas: boolean;

  /**
   * The array of coupons
   * @type {ICoupon[]}
   */
  coupon?: ICoupon[];

  /**
   * Selected store
   * @type {INearestStore}
   */
  selectedStore?: INearestStore;

  /**
   * Customer name
   * @type {string}
   */
  customerName?: string;
}
