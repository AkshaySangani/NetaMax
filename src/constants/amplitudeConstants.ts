export const HOME_VIEWED = "Home_Viewed";
export const PRODUCT_ADDED = "Product_Added";
export const CATEGORY_CLICKED = "Category_Clicked";
export const BANNER_CLICKED = "Banner_Clicked";
export const PRODUCT_DETAIL_VIEWED = "Product_Detail_Viewed";
export const CART_PAGE_VIEWED = "Cart_Page_Viewed";
export const NAME_ADDED = "Name_Added";
export const PHONE_ADDED = "Phone_Added";
export const ZIPCODE_PAGE_VIEWED = "Zipcode_Page_Viewed";
export const ZIPCODE_VERIFIED = "Zipcode_Verified";
export const LOGIN_COMPLETED = "Login_Completed";
export const ORDER_COMPLETED = "Order_Completed";
export const CHECKOUT_PAGE_VIEWED = "Checkout_Page_Viewed";
export const REFERRAL_CODE_CLICKED = "Referral_Code_Clicked";
export const CATEGORY_VER_TODOS_CLICKED = "Category_Ver_Todos_Clicked";
export const SEARCH_VIEW_MORE = "Search_View_More";
export const SEARCHBAR_CLICKED = "Searchbar_Clicked";
export const PROMO_SCROLLED = "Promo_Scrolled";
export const CATEGORY_SCROLLED = "Category_Scrolled";
export const BANNER_SCROLLED = "Banner_Scrolled";
export const CATEGORY_PRODUCT_SCROLLED = "Category_Product_Scrolled";
export const SUPPORT_CHAT_CLICKED = "Support_Chat_Clicked";
export const OTP_REQUESTED = "OTP_Requested";
export const OTP_PAGE_VIEWED = "OTP_Page_Viewed";
export const OTP_ERROR_TOASTED = "OTP_Error_Toasted";
export const OTP_RESEND_CODE = "OTP_Resend_Code";
export const STORE_SELECTED_CLICKED = "Store_Selected_Clicked";
export const SEARCH_ZIPCODE = "Search_ZipCode";
export const HOME_IS_YOUNGER_USER = "Home_Is_Younger_User";
export const NODE_PRODUCT_SCROLLED = "Node_Product_Scrolled";
export const NODE_VER_TODOS_CLICKED = "Node_Ver_Todos_Clicked";
export const HOME_ARROW_CLICKED = "Home_Arrow_Clicked";

// Admin shops event constants
export const SA_LOGIN_PAGE_VIEWED = "SA_Login_Page_Viewed";
export const SA_OTP_REQUESTED = "SA_otp_requested";
export const SA_LOGIN_COMPLETED = "SA_login_completed";
export const SA_OTP_TOAST_ERROR = "SA_otp_toast_error";
export const SA_LOGIN_FAILED = "SA_login_failed";
export const SA_INVOICES_PAGE_VIEWED = "SA_invoices_page_viewed";
export const SA_INVOICES_PAGE_SCROLLED = "SA_invoices_page_scrolled";
export const SA_INFO_PAGE_VIEWED = "SA_Info_Page_Viewed";
export const SA_SHARE_STORELINK_BANNER_CLICKED = "SA_Share_StoreLink_Banner_Clicked";
export const SA_CLIENTS_PAGE_VIEWED = "SA_Clients_Page_Viewed";
export const SA_PHONE_ADDED = "SA_Phone_Added";
export const SA_OTP_RESEND_CODE = "SA_OTP_Resend_Code";
export const SA_INVOICE_CLICKED = "SA_Invoice_Clicked";
export const SA_TRACK_LAST_ORDER_CLICKED = "SA_Track_Last_Order_Clicked";
//TODO: Check with business the grammar of 'Especific'
export const SA_TRACK_SPECIFIC_ORDER_CLICKED = "SA_Track_Especific_Order_Clicked";
export const SA_VIEW_ORDERS_CLICKED = "SA_View_Orders_Clicked";
export const SA_ORDERS_PAGE_VIEWED = "SA_Orders_Page_Viewed";
export const SA_VIEW_ORDER_DETAIL_CLICKED = "SA_View_Order_Detail_Clicked";
export const SA_ORDER_DETAIL_VIEWED = "SA_Order_Detail_Viewed";
export const SA_LOGOUT_CLICKED = "SA_Logout_Clicked";
export const SA_TOKEN_PAGE = "SA_Token_Page";
export const SA_TOKEN_REDEEMED = "SA_Token_Redeemed";
export const SA_ORDER_DELIVERED_PAGE = "SA_Order_Delivered_Page";
export const SA_TOKEN_ERROR_FOUND = "SA_Token_Error_Found";
export const SA_TOKEN_ALTERNATIVE_VALIDATION = "SA_Token_Alternative_Validation";
export const SA_FORCE_TOKEN_VALIDATION = "SA_Force_Token_Validation";
export const SA_DATE_SELECTED = "SA_Date_Selected";
export const SA_ACTUALIZAR_CLICKED = "SA_Actualizar_Clicked";
export const SA_COPIARLIGA_CLICKED = "SA_CopiarLiga_Clicked";
export const SA_HOME_PAGE_VIEWED = "SA_Home_Page_Viewed";
export const SA_VENTAS_PAGE_VIEWED = "SA_Ventas_Page_Viewed";
export const SA_ENDCLIENT_WA_CLICKED = "SA_EndClient_WA_Clicked";
export const SA_COMPARTIR_LIGA_CLICKED = "SA_Compartir_Liga_Clicked";
export const SA_TRACK_ORDER_CLICKED = "SA_Track_Order_Clicked";
export const SA_AYUDA_CLICKED = "SA_Ayuda_Clicked";
export const SA_VOLVER_BUTTON_CLICKED = "SA_Volver_Button_Clicked";
export const SA_DESCARGAR_BUTTON_CLICKED = "SA_Descargar_Button_Clicked";

// GanaDinero event constants
export const MONETAS_GMV_BUTTON_CLICKED = "Monetas_GMV_Quiero_comprar_Button_Clicked";
export const TAB_GANADINERO_BUTTON_CLICKED = "Tab_GanaDinero_Button_Clicked";
export const HOME_NO_MONETAS_BUTTON_CLICKED = "Home_No_Monetas_Button_Clicked";
export const HOME_MONETAS_NOT_LOGGED_BUTTON_CLICKED = "Home_Monetas_Not_Logged_Button_Clicked";
export const DAILY_REWARD_BUTTON_CLICKED = "Daily_Reward_Button_Clicked";
export const DAILY_REWARD_INGRESAR_DATOS_BUTTON_CLICKED =
  "Daily_Reward_Ingresar_Datos_Button_Clicked";
export const MONETAS_SKU_QUIERO_COMPRAR_BUTTON_CLICKED =
  "Monetas_SKU_Quiero_comprar_Button_Clicked";
export const MONETAS_GMV_PROGRESS_BAR_COMPLETED = "Monetas_GMV_Progress_Bar_Completed";
export const MONETAS_GMV_SUCCESS = "Monetas_GMV_Success";
export const MONETAS_SKU_BUTTON_CLICKED = "Monetas_Sku_Activated";
export const MONETAS_SKU_SUCCESS = "Monetas_SKU_Success";
export const MONETAS_SKU_PROGRESS_BAR_COMPLETED = "Monetas_SKU_Progress_Bar_Completed";
export const MONETAS_IS_SKU_CHALLENGE = "Is_Monetas_SKU_Challenge";
export const MONETAS_CATEGORY_BUTTON_CLICKED = "Monetas_Category_Activated";
export const MONETAS_CATEGORY_PROGRESS_BAR_COMPLETED = "Monetas_Category_Progress_Bar_Completed";
export const MONETAS_CATEGORY_SUCCESS = "Monetas_Category_Success";
export const MONETAS_IS_CATEGORY_CHALLENGE = "Is_Monetas_Category_Challenge";

// MisCompras event constants
export const TAB_MISCOMPRAS_BUTTON_CLICKED = "Tab_MisCompras_Button_Clicked";
export const TAB_MISCOMPRAS_VIDEO_PlAYED = "Tab_MisCompras_Video_Played";
export const TAB_MISCOMPRAS_IR_A_COMPRAR = "Tab_MisCompras_Ir_A_Comprar";
export const TAB_MISCOMPRAS_VER_DETALLE_PEDIDO = "Tab_MisCompras_Ver_Detalle_Pedido";
export const TAB_MISCOMPRAS_VER_DETALLE_PEDIDO_WHATSAPP_TIEND =
  "Tab_MisCompras_Ver_Detalle_Pedido_Whatsapp_Tiend";
export const TAB_MISCOMPRAS_VER_DETALLE_PEDIDO_VER_MAS =
  "Tab_MisCompras_Ver_Detalle_Pedido_Ver_Mas";

//MiCuenta event constants
export const MI_CUENTA_PAGE_VIEWED = "Mi_Cuenta_Page_Viewed";
export const MI_CUENTA_INGRESAR_DATOS_CLICKED = "Mi_Cuenta_Ingresar_Datos_Clicked";
export const MI_CUENTA_ERES_NUEVO_CLICKED = "Mi_Cuenta_Eres_Nuevo_Clicked";
export const MI_CUENTA_MIS_COMPRAS_PAGE_VIEWED = "Mi_Cuenta_Mis_Compras_Page_Viewed";
export const MIS_COMPRAS_NO_PURCHASES_PAGE_VIEWED = "Mis_Compras_No_Purchases_Page_Viewed";
export const MIS_COMPRAS_NO_PURCHASES_BUTTON_CLICKED = "Mis_Compras_No_Purchases_Button_Clicked";

// Checkout event constants
export const CHO_ENTER_ADDRESS_PAGE = "Enter_Address_Page";
export const CHO_STORES_RESULTS_LISTING_PAGE = "Stores_Results_Listing_Page";
export const CHO_ORDER_SUMMARY_PAGE = "Order_Summary_Page";

export enum TrackerApp {
  Amplitude = "amplitude",
  Segment = "segment",
}
