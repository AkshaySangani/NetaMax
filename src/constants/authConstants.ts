export const LOGIN_URL = "login/";
export const REFRESH_TOKEN_URL = "refresh/";
export const OTP_CODE_URL_SEC = "security/generateOtp";
export const GET_CUSTOMER_INFO_URL = "customer/me";
export const ZIP_CODE_URL = "zipcode_customer/";
export const OTP_VALIDATE_CODE_URL = "security/validateOtp";
export const INVALID_CODE = "Código inválido";
export const PLATFORM_OTP_LOGIN = "codigo_verificacion_usuario";
export enum OtpChannel {
  "WHATSAPP" = "whatsapp",
}
