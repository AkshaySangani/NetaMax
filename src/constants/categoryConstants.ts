export const GET_ALL_CATEGORIES_API_URL = "category/";
export const CATEGORY_TITLE = "Secciones";
export const DISCOUNT_TITLE = "Destacados";
export const MAX_CATEGORIES_PER_SLIDE_MOBILE = 2.3;
export const MAX_CATEGORIES_PER_SLIDE_TABLET = 4.3;
export const MAX_CATEGORIES_PER_SLIDE_DESKTOP = 6.3;
export const CATEGORY_SLIDE_SPACE_BETWEEN_ITEMS_MOBILE = 0;
export const CATEGORY_SLIDE_SPACE_BETWEEN_ITEMS_TABLET = 0;
export const CATEGORY_SLIDE_SPACE_BETWEEN_ITEMS_DESKTOP = 0;

export const categoryColors: Record<string, { color: string; name: string }> = {
  "70": { color: "#FFFDF5", name: "Lácteos y Refrigerados" },
  "73": { color: "#FFF9F9", name: "Canasta Básica" },
  "7": { color: "#FFFAEC", name: "Frutas y Verduras" },
  "3": { color: "#F3FFFC", name: "Despensa" },
  "68": { color: "#FFFDE9", name: "Licores y Cigarros" },
  "15": { color: "#F4F2FF", name: "Bebidas" },
  "69": { color: "#FAFFF2", name: "Dulces y Botanas" },
  "2": { color: "#FFFEF1", name: "Cuidado Personal" },
  "14": { color: "#FFEDED", name: "Mascotas" },
  "74": { color: "#FDFFF7", name: "Bebes y Niños" },
  "78": { color: "#F6FFFD", name: "Belleza" },
};

export const bannerNames = [
  "300.png",
  "301.png",
  "302.png",
  "305.png",
  "325.png",
  "327.png",
  "328.png",
];
