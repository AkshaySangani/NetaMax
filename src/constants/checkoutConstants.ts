export enum CheckoutWizardSteps {
  BASKET = 1,
  NAME_AND_TERM_CONDITIONS = 2,
  PHONE_NUMBER = 3,
  VERIFICATION_CODE = 4,
  POSTAL_CODE = 5,
  STORE_BY_DEFAULT = 6,
  FIND_STORE = 7,
  ORDER_REVIEW = 8,
  ORDER_SUCCESS = 9,
}

export const loginCheckoutWizardSteps = [
  CheckoutWizardSteps.BASKET,
  CheckoutWizardSteps.NAME_AND_TERM_CONDITIONS,
  CheckoutWizardSteps.PHONE_NUMBER,
  CheckoutWizardSteps.VERIFICATION_CODE,
];

export enum OrderStatus {
  DRAFT = "draft",
  SUBMITTED = "submitted",
  REJECTED = "rejected",
}

export const ASSISTANCE_PHONE_NUMBER = "5215545439866";

export const PLACE_ORDER_URL = "order/";

export const PLACE_ORDER_URL_V2 = `${process.env.NEXT_PUBLIC_API_ROOT_URL}/api/v2/order`;

export const COUNTDOWN_TIMER_SECONDS = 45;

export const OTP_CODE_LENGTH = 4;

export const SHARE_DISCOUNT_CODE_TEMPLATE =
  "Te *regalo $20 pesos* para que compres en *Neta* usando mi código:{código} Encontrarás los mejores productos a los mejores precios de la ciudad en la liga. Entra y chécala en:{liga} *No olvides ingresar el código de descuento al finalizar el pedido* (Solo aplica para clientes nuevos en su primera orden)";

export const INVALID_CUSTOMER_MESSAGE =
  "Lo sentimos, tu orden no ha sido procesada debido a que presenta una conducta inapropiada según nuestros Términos y condiciones de Uso, contacta a soporte si consideras que hay un error.";

export const PLACES_STATUS_OK = "OK";

export const PLACES_STATUS_ZERO_RESULTS = "ZERO_RESULTS";

export const NUMBER_WITH_PREFIX_LENGTH = 12;

export const NUMBER_WITHOUT_PREFIX_LENGTH = 10;

export const MIS_COMPRAS_ROUTE = "/mi-cuenta/mis-compras";

export const MI_CUENTA_ROUTE = "/mi-cuenta";
