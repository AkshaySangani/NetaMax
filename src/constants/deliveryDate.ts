export const NEXT_DAY = "Llega en 1 dia";
export const BETWEEN_THREE_FIVE_DAYS = "Llega entre 3 y 5 dias";
export const BETWEEN_SIX_NINE_DAYS = "Llega entre 6 y 9 dias";
