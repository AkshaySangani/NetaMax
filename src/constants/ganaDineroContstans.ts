export const GET_COINS_BY_CUSTOMER_ID = "customer-neta-wallet";

export const GMV_ID = 1;
export const EVENTS_ID = 2;
export const DISCOVERY_ID = 3;
export const REFERALS_ID = 4;
export const SURVEY_ID = 5;

// TODO: this might change when having real challenges code names or ids
export const SKU_NAME = "SKU";
export const CATEGORY_NAME = "Category";
