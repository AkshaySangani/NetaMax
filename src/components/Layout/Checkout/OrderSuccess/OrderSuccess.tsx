import React, { ReactElement } from "react";

import { ASSISTANCE_PHONE_NUMBER } from "constants/checkoutConstants";
import CheckoutCheckmark from "styled/icons/CheckoutFlow/CheckMark";
import WhatsappIcon from "styled/icons/WhatsappIcon";

import { Box, Button, Container, Flex, Link, Text } from "@chakra-ui/react";

import { DeliverySite } from "../DeliverySite/DeliverySite";
import { PurchaseSummary } from "../PurchaseSummary/PurchaseSummary";
import { IOrderSuccessProps } from "./IOrderSuccessProps";

/**
 *  The order success screen component
 * @param {IOrderSuccessProps} props The component props
 * @returns {ReactElement} The order success screen component
 */
export const OrderSuccess = (props: IOrderSuccessProps): ReactElement => {
  const {
    formattedPhone,
    couponSumSuccess,
    totalSumSuccess,
    partialSumSuccess,
    monetasSumSuccess,
    coupon,
    payWithMonetas,
    onReferralShareButtonClick,
    selectedStore,
    shareDiscountCodeText,
    customerName,
  } = props;

  return (
    <Box>
      <Flex
        position="absolute"
        top="0"
        left="0"
        direction="column"
        background="#3CE88B"
        justifyContent="space-evenly"
        height="250px"
        zIndex="10"
      >
        <Container marginY="28px">
          <Box>
            <CheckoutCheckmark />
          </Box>
          <Box marginTop="18px" paddingBottom="10px">
            <Text fontSize="32px" fontWeight="800" lineHeight="40px">
              ¡Tu pedido ha sido confirmado!
            </Text>
            <Text marginTop="4px" fontSize="14px">
              {customerName || ""} {`${customerName ? "t" : "T"}`}e hemos enviado un whatsapp
              confirmando el pedido, al número <b>{formattedPhone}</b>
            </Text>
          </Box>
        </Container>
      </Flex>
      <Box paddingTop="180px">
        <DeliverySite selectedStore={selectedStore} fromSuccess />
        <Box mt="40px">
          <Text fontSize="18px" fontWeight="800" lineHeight="40px">
            ¡Regala con Neta!
          </Text>
          <Text fontSize="14px" opacity="0.7">
            Invita a tus amigos y tendrán un descuento de 20 pesos en su primera orden.
          </Text>
          <Link
            href={`https://wa.me/?text=${encodeURIComponent(shareDiscountCodeText)}`}
            target="_blank"
            passHref
          >
            <Button
              marginTop="8px"
              border="1px solid #353535"
              borderRadius="48px"
              paddingX="16px"
              paddingY="6px"
              background="white"
              minWidth="100px"
              onClick={onReferralShareButtonClick}
            >
              <Text fontSize="14px" fontWeight="600">
                Invitar amigos
              </Text>
            </Button>
          </Link>
        </Box>
        <Box marginTop="42px" marginBottom="8px">
          <Text fontSize="18px" fontWeight="800" lineHeight="40px">
            ¿Necesitas ayuda con tu pedido?
          </Text>
          <Text fontSize="14px" opacity="0.7">
            Si deseas cancelar o editar tu pedido comunicate con nosotros via whatsapp par ayudarte
          </Text>
        </Box>
        <Link href={`https://wa.me/${ASSISTANCE_PHONE_NUMBER}`} target="_blank" passHref>
          <Button borderRadius="24px" colorScheme="whatsapp">
            <WhatsappIcon />
            <Text color="white" fontSize="16px" marginLeft="8px">
              Necesito ayuda con mi pedido
            </Text>
          </Button>
        </Link>
        <Text fontSize="18px" fontWeight="800" lineHeight="40px" marginTop="32px">
          Resumen del pedido
        </Text>
        <PurchaseSummary
          discountAmount={couponSumSuccess}
          totalAmount={totalSumSuccess}
          totalNetPrice={partialSumSuccess}
          walletAmount={monetasSumSuccess}
          coupon={coupon}
          payWithMonetas={payWithMonetas}
        />
      </Box>
    </Box>
  );
};
