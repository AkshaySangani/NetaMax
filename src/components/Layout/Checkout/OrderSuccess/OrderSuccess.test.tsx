import { render, cleanup, fireEvent, screen } from "@testing-library/react";
import { OrderSuccess } from "./OrderSuccess";
import { IOrderSuccessProps } from "./IOrderSuccessProps";
import { ASSISTANCE_PHONE_NUMBER } from "constants/checkoutConstants";

describe("OrderSuccess Component", () => {
  const onReferralShareButtonClick = jest.fn();
  let mock: IOrderSuccessProps;

  beforeEach(() => {
    mock = {
      shareDiscountCodeText: "COUPENCODE",
      onReferralShareButtonClick,
      showCoinAwardedCard: true,
      formattedPhone: "1111111111",
      partialSumSuccess: 60,
      couponSumSuccess: 10,
      monetasSumSuccess: 20,
      totalSumSuccess: 30,
      payWithMonetas: true,
    };
  });

  afterEach(() => {
    cleanup();
    jest.clearAllMocks();
  });

  test("OrderSuccess should be render correctly", () => {
    const { container } = render(<OrderSuccess {...mock} />);
    expect(container).toMatchSnapshot();
  });

  test("Invitar amigos button working properly", () => {
    const { getByText } = render(<OrderSuccess {...mock} />);
    fireEvent.click(getByText("Invitar amigos"));
    expect(onReferralShareButtonClick).toBeCalledTimes(1);
  });

  test("Link should be render properly", () => {
    render(<OrderSuccess {...mock} />);
    expect(screen.getByText("Necesito ayuda con mi pedido").closest("a")).toHaveAttribute(
      "href",
      `https://wa.me/${ASSISTANCE_PHONE_NUMBER}`
    );
  });
});
