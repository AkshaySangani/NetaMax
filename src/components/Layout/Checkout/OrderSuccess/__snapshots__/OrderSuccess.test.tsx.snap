// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderSuccess Component OrderSuccess should be render correctly 1`] = `
<div>
  <div
    class="css-0"
  >
    <div
      class="css-1poqpeo"
    >
      <div
        class="chakra-container css-cs0kzy"
      >
        <div
          class="css-0"
        >
          <svg
            fill="none"
            height="36"
            viewBox="0 0 40 36"
            width="40"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M38.2533 0.602678C36.4483 -0.369247 34.435 1.50518 33.2548 2.61595C30.5474 5.25403 28.2565 8.30865 25.6877 11.0856C22.8414 14.1402 20.2033 17.1948 17.2875 20.1801C15.6213 21.8463 13.8163 23.6513 12.7056 25.734C10.2063 23.3041 8.05422 20.666 5.27729 18.514C3.26402 16.9867 -0.0682945 15.8759 0.0011287 19.5553C0.139975 24.3456 4.37479 29.483 7.49883 32.7457C8.81787 34.1342 10.5535 35.5921 12.5667 35.6615C14.9965 35.8004 17.4958 32.8846 18.9537 31.2878C21.5224 28.5109 23.6051 25.3868 25.9654 22.5405C29.02 18.7917 32.1441 15.1121 35.1293 11.2938C37.0037 8.93346 42.9047 3.1018 38.2533 0.602678ZM3.05564 19.2776C2.98621 19.2776 2.91679 19.2776 2.77794 19.3469C2.50025 19.2776 2.29198 19.2081 2.01429 19.0692C2.22256 18.9304 2.56967 18.9998 3.05564 19.2776Z"
              fill="#353535"
            />
          </svg>
        </div>
        <div
          class="css-ouzjzl"
        >
          <p
            class="chakra-text css-1d7w6d6"
          >
            ¡Tu pedido ha sido confirmado!
          </p>
          <p
            class="chakra-text css-crpoxa"
          >
            
             
            T
            e hemos enviado un whatsapp confirmando el pedido, al número 
            <b>
              1111111111
            </b>
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-sgpgrv"
    >
      <div
        class="css-k8wr52"
      >
        <div
          class="css-1siwe0b"
        >
          <p
            class="chakra-text css-okz3aw"
          >
            Sitio de entrega
          </p>
          <p
            class="chakra-text css-8whdou"
          >
            Esta es la tiendita que elegiste para la entrega
          </p>
          <div
            class="css-mifb2i"
          >
            <div
              class="css-2jtlu4"
            >
              <div
                class="css-qim7r6"
              >
                <img
                  class="chakra-image css-34k09d"
                  src="/assets/images/store.png"
                />
              </div>
            </div>
            <div
              class="css-12w7o40"
            >
              <p
                class="chakra-text css-1mzljxq"
              />
              <p
                class="chakra-text css-w6om4s"
              />
              <a
                class="chakra-link css-1okj0fn"
                decoration="underline"
                href="https://www.google.com/maps/place/undefined,undefined"
                target="_blank"
              >
                Ver ubicación
              </a>
              <p
                class="chakra-text css-1vksuc8"
              >
                Contacta con la tiendita
              </p>
              <a
                class="chakra-link css-1okj0fn"
                decoration="underline"
                href="https://wa.me/52undefined"
                target="_blank"
              >
                Ir al whatsapp de la tiendita
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="css-y1gt6f"
      >
        <p
          class="chakra-text css-gth86c"
        >
          ¡Regala con Neta!
        </p>
        <p
          class="chakra-text css-cyjlo4"
        >
          Invita a tus amigos y tendrán un descuento de 20 pesos en su primera orden.
        </p>
        <a
          class="chakra-link css-0"
          href="https://wa.me/?text=COUPENCODE"
          target="_blank"
        >
          <button
            class="chakra-button css-lh4mb9"
            type="button"
          >
            <p
              class="chakra-text css-1c1e297"
            >
              Invitar amigos
            </p>
          </button>
        </a>
      </div>
      <div
        class="css-q34kyj"
      >
        <p
          class="chakra-text css-gth86c"
        >
          ¿Necesitas ayuda con tu pedido?
        </p>
        <p
          class="chakra-text css-cyjlo4"
        >
          Si deseas cancelar o editar tu pedido comunicate con nosotros via whatsapp par ayudarte
        </p>
      </div>
      <a
        class="chakra-link css-0"
        href="https://wa.me/5215545439866"
        target="_blank"
      >
        <button
          class="chakra-button css-48w0h7"
          type="button"
        >
          <svg
            fill="none"
            height="28"
            viewBox="0 0 28 28"
            width="28"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M23.4485 4.54189C20.9414 2.04421 17.6079 0.668015 14.056 0.666748C6.73774 0.666748 0.781261 6.59415 0.778078 13.88C0.776804 16.2092 1.38862 18.4825 2.55051 20.486L0.666664 27.3334L7.70545 25.496C9.64468 26.549 11.8284 27.1034 14.0503 27.1041H14.056C14.0567 27.1041 14.0554 27.1041 14.056 27.1041C21.3737 27.1041 27.3308 21.176 27.3333 13.8902C27.3346 10.3597 25.955 7.03958 23.4485 4.54189ZM14.056 24.8725H14.0516C12.0716 24.8719 10.1292 24.3422 8.43441 23.3417L8.03141 23.1035L3.85436 24.1939L4.96914 20.1407L4.70684 19.7251C3.60225 17.9763 3.01844 15.9551 3.01972 13.88C3.02226 7.82462 7.97284 2.89768 14.0605 2.89768C17.0082 2.89895 19.7789 4.04261 21.8626 6.11895C23.9464 8.19465 25.093 10.9546 25.0917 13.8889C25.0891 19.9455 20.1386 24.8725 14.056 24.8725ZM20.1093 16.647C19.7776 16.4816 18.1465 15.6833 17.8422 15.573C17.5379 15.4628 17.3169 15.4076 17.096 15.7384C16.8751 16.0691 16.2391 16.813 16.0455 17.0328C15.852 17.2533 15.6585 17.2806 15.3268 17.1152C14.9951 16.9498 13.9261 16.6014 12.6586 15.4767C11.6724 14.6011 11.0065 13.5201 10.8129 13.1894C10.6194 12.8586 10.7926 12.68 10.9581 12.5159C11.1071 12.3676 11.2898 12.13 11.456 11.9374C11.6221 11.7448 11.6769 11.6066 11.7876 11.3868C11.8984 11.1663 11.843 10.9737 11.7603 10.8083C11.6775 10.6429 11.0141 9.01771 10.7372 8.35685C10.4679 7.71311 10.1941 7.80054 9.99102 7.78977C9.79748 7.78027 9.57656 7.77837 9.35501 7.77837C9.13409 7.77837 8.77438 7.86074 8.47007 8.19148C8.16575 8.52222 7.30882 9.3212 7.30882 10.9458C7.30882 12.571 8.49745 14.1411 8.66361 14.3616C8.82978 14.5821 11.0033 17.9167 14.3311 19.3474C15.1224 19.6877 15.7406 19.8911 16.2225 20.0431C17.0171 20.2947 17.7403 20.2592 18.312 20.1743C18.9493 20.0792 20.2748 19.3753 20.5511 18.6042C20.8274 17.8331 20.8274 17.1716 20.7447 17.0341C20.6625 16.8947 20.441 16.8124 20.1093 16.647Z"
              fill="white"
              fill-rule="evenodd"
            />
          </svg>
          <p
            class="chakra-text css-bsjcwp"
          >
            Necesito ayuda con mi pedido
          </p>
        </button>
      </a>
      <p
        class="chakra-text css-10d023v"
      >
        Resumen del pedido
      </p>
      <div
        class="css-wmz062"
      >
        <div
          class="css-z7mtfw"
        >
          <p
            class="chakra-text css-0"
          >
            Suma parcial
          </p>
          <p
            class="chakra-text css-1nhysjy"
          >
            $60.00
             
          </p>
        </div>
        <div
          class="css-z7mtfw"
        >
          <p
            class="chakra-text css-0"
          >
            Envío
          </p>
          <p
            class="chakra-text css-1nhysjy"
          >
            Gratis
          </p>
        </div>
        <div
          class="css-z7mtfw"
        >
          <div
            class="css-k008qs"
          >
            <p
              class="chakra-text css-0"
            >
              Monetas
            </p>
            <img
              class="chakra-image css-1isemmb"
              src="/assets/images/icons/dolar.png"
            />
          </div>
          <p
            class="chakra-text css-fa9nx4"
          >
            - 
            $20.00
          </p>
        </div>
        <div
          class="css-eofv09"
        >
          <hr />
        </div>
        <div
          class="css-z7mtfw"
        >
          <p
            class="chakra-text css-ha3xn3"
          >
            Total a pagar
          </p>
          <p
            class="chakra-text css-1nhysjy"
          >
            $30.00
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
